src/NServiceBus.Core.Tests.x86/When_running_under_a_32Bit_process.cs

```diff
@@ -1,22 +0,0 @@
-﻿namespace NServiceBus.Core.Tests.x32
-{
-    using System.Linq;
-    using Hosting.Helpers;
-    using NUnit.Framework;
-
-    [TestFixture]
-    public class When_running_under_a_32Bit_process
-    {
-        [Test]
-        public void Should_skip_x64_assemblies()
```

Comment:
where does that scenario come from? How likely is it for someone to run into this issue? By removing that code in the assembly scanner, we would just point out to the user that he is using incompatible assemblies instead of silently ignoring them?